src/services/http.ts

```diff
@@ -53,7 +53,12 @@ const patch = <TResponse>(
 ): Promise<TResponse> =>
   makeRequest(endpoint, 'PATCH', JSON.stringify(body), setContentTypeJSON(headers));
 
-const del = <TResponse>(endpoint: string): Promise<TResponse> => makeRequest(endpoint, 'delete');
+const del = <TResponse>(
+  endpoint: string,
+  body: Object = '',
+  headers = new Headers(),
+): Promise<TResponse> =>
+  makeRequest(endpoint, 'delete', JSON.stringify(body), setContentTypeJSON(headers));
```

Comment:
@EjPlatzer Is this good enough for `nullable` as you mentioned? Or do you want it to be `?` nullable?